src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/CustomLineCap.cs

```diff
@@ -14,7 +14,7 @@ public partial class CustomLineCap : MarshalByRefObject, ICloneable, IDisposable
 #endif
 
         // Handle to native line cap object
-        internal SafeCustomLineCapHandle nativeCap = null;
+        internal SafeCustomLineCapHandle nativeCap = null!;
 
         private bool _disposed = false;
 
```

Comment:
Cannot comment on the row 24-28 parameters `GraphicsPath fillPath, GraphicsPath strokePath` are allowing null